### DISCUSSION TITLE
SEB does not find exe (win) or .app (OS X) defined in the applications tab

### DISCUSSION THREAD

**Discussion by jpmaroco** [SafeExamBrowser/seb-win-refactoring#375] [Q&A]
Dear all,
After reading the manuals for both Win and OS X versions of SEB 3.3 Configuration Tool, I can't get it to find and run the SPSS Statistics (stats.exe for windows; SPSS Statistics.app for OSX). Here is what I did:

1. First just typed "stats.exe" in the executable box for Win, it did not work (i.e., did not find the stats.exe, altough SPSS statistics was installed with default options and I can run in the cmd box). It did work when I added the full path:
![image](https://user-images.githubusercontent.com/89992415/157666760-5a35e2e6-89b0-44fb-85c3-4ad54d146353.png)

2. For the Mac OS X I also just added the "SPSS Statistics.app" to the Executable box. Did not worked.
2.1. Following  the example from Windows, I added the full path:
![image](https://user-images.githubusercontent.com/89992415/157666972-417beacb-f1f3-46dc-a90c-366e79adb90f.png)
It does not work either.

Is this a bug? or am I doing something wrong with the Mac Configuration?

Thanks for any insights on how to get the SPSS Statistics app running with SEB.

**Comment by dbuechel** [comment]
For Windows, we'd recommend to use the button "Choose Application...", as that way various fields like original name and path should automatically be filled in correctly. When the application still doesn't start during a session, then you should be able to find the reason in either the client application log (to be found under `%LocalAppData%\SafeExamBrowser\Logs`) or in the live application log during a session (which you can enable under tab "Security"). The macOS version of SEB does to my knowledge not (yet) support third-party applications.

**Reply by soutienti** [reply]
hello 
Do you know when the MacOS version of SEB will support third-party applications ? 
Thanks !

**Reply by dbuechel** [reply]
From what I know there are no specific plans at the moment. But please do feel free to ask the question in the macOS repository here where you'll get the correct and specific response.

**Reply by soutienti** [reply]
thanks for your answer, I will try. :)

**Reply by soutienti** [reply]
hello, I try to create a topic about this subject
https://github.com/SafeExamBrowser/seb-mac/discussions/375
but I don't have receive any reply. do I put in the good place ? 
thanks for your help.
